language/move-core/types/src/account_address.rs

```diff
@@ -119,13 +119,13 @@ impl std::ops::Deref for AccountAddress {
 
 impl fmt::Display for AccountAddress {
     fn fmt(&self, f: &mut fmt::Formatter) -> std::fmt::Result {
-        write!(f, "{:#x}", self)
+        write!(f, "{}", self.to_hex_literal())
```

Comment:
The `display` also affects the json serialize format, so we decide to use the short format in all places?